templates/service-template.yml

```diff
@@ -595,6 +595,11 @@ parameters:
   description: As a user, one can create up to N defined max developer instances if they do not have quota to create standard instances.
   value: "1"
 
+- name: OBSERVABILITY_ENABLE_CLOUDWATCHLOGGING
+  displayName: Enable Observability CloudWatch Logging
+  description: Enables Observability related AWS CloudWatch Logging. If false, logs from the data plane won't be sent to CloudWatch through Observability
```

Comment:
related to https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1527#discussion_r1091786583

Reply:
Answered there